_topics/en/countersign.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Countersign
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - P2P Network Protocol
+  - Security Enhancements
+  - Privacy Enhancements
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+excerpt: >
+  **Countersign** is the idea for a protocol that will allow a client and
+  server who have each others public keys to negotiate authentication
+  without either revealing any identifying information to third parties.
```

Comment:
'without either party revealing any identifying information.' (without a noun, either attaches to the verb revealing on initial parse)